src/routes/popup/arns/ArNSPurchaseSuccessView.tsx

```diff
@@ -0,0 +1,112 @@
+import { ButtonV2 } from "@arconnect/components";
```

Comment:
Always use `@arconnect/components-rebrand` instead of `@arconnect/components`. There might still be some usage of `@arconnect/components` in our codebase which will be removed soon.